### PR TITLE
zh, en: update release note for v2.0.6

### PR DESCRIPTION
**Pull request by GMHDBJD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/releases/2.0.6.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# DM 2.0.6 Release Notes
+
+发版日期：2021 年 8 月 13 日
+
+DM 版本：2.0.6
+
+## Bug 修复
+
+- 修复乐观模式下 DDL infos 和 upstream table 的元数据不一致导致 DM-master panic 的问题[#1971](https://github.com/pingcap/dm/pull/1971)
```
Comment (lance6716) [inline zh/releases/2.0.6.md:13]: 感觉 “乐观模式” 这个名字太宽泛，希望叫 “乐观 shard DDL”、“乐观协调”之类的

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Review by Liuxiaozhen12 — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Pull request merged**